release-notes/unreleased/2850-POI-filter-button-shines-through.yml

```diff
@@ -0,0 +1,6 @@
+issue_key: 2850
+show_in_stores: true
```

Comment:
Since this is only a web issue, it doesn't need to be shown in the stores :)

Reply:
Accordingly, no German translation is necessary :) We only need German translations for ios/android issues that are shown in the stores.